internal/cli/atlas/privateendpoints/gcp/watch.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gcp
+
+import (
+	"context"
+
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/cli/require"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+type WatchOpts struct {
+	cli.GlobalOpts
+	cli.WatchOpts
+	id    string
+	store store.PrivateEndpointDescriber
+}
+
+func (opts *WatchOpts) initStore(ctx context.Context) func() error {
+	return func() error {
+		var err error
+		opts.store, err = store.New(store.AuthenticatedPreset(config.Default()), store.WithContext(ctx))
+		return err
+	}
+}
+
+func (opts *WatchOpts) watcher() (bool, error) {
+	result, err := opts.store.PrivateEndpoint(opts.ConfigProjectID(), provider, opts.id)
+	if err != nil {
+		return false, err
+	}
+	return result.Status == "AVAILABLE" || result.Status == "FAILED", nil
+}
+
+func (opts *WatchOpts) Run() error {
+	if err := opts.Watch(opts.watcher); err != nil {
+		return err
+	}
+
+	return opts.Print(nil)
+}
+
+// mongocli atlas privateEndpoints gcp watch <privateEndpointId> [--projectId projectId].
+func WatchBuilder() *cobra.Command {
+	opts := &WatchOpts{}
+	cmd := &cobra.Command{
+		Use:   "watch <privateEndpointId>",
+		Short: "Watch the specified GCP private endpoint to detect changes in the endpoint's state.",
```

Comment:
I think we use `` instead of double quotes for all of these.

Reply:
we use **`** for multi-line strings